### DISCUSSION TITLE
dot-once....?

### DISCUSSION THREAD

**Discussion by edwardsmarkf** [katspaugh/wavesurfer.js#2875] [Q&A] [answered]
hello - 

all of my regular readers know i am not the brightest of the wavesurfer minions.

in this link  https://wavesurfer-js.org/examples/#basic.js  i see:
```
wavesurfer.once('interaction', () => {
  wavesurfer.play()
})
```
what is `.once(`?   is this a typo?   is this supposed to be `.on(`?

somehow i feel like a beginning art student questioning the brushstroke that Leonardo Da Vinci used in the Mona Lisa painting.

**Comment by katspaugh** [comment, marked as answer]
Good question! `Once` is the same as `on` but it will fire only… once. 😄

**Reply by edwardsmarkf** [reply]
awww i should have looked that one up...   thanks for the reply.
